Hadrons/Modules/MContraction/Meson.hpp

```diff
@@ -85,6 +85,8 @@ class TMeson: public Module<MesonPar>
                                         Gamma::Algebra, gamma_snk,
                                         Gamma::Algebra, gamma_src,
                                         std::vector<Complex>, corr);
+        Result() {};
+        Result(uint16_t dummy) {};
```

Comment:
Dummy single-arg constructor required here due to the use of `envCreate` in `setup`

Reply:
`enCreate` should not for you to do that, the variadic macro argument can be empty e.g. `MACRO(a,b,)`

Reply:
Constructors removed and macro called fixed in 6a34ab619a5d7cca1bb6abcaf3aac014e8fba517